src/routes/index.js

```diff
@@ -3,7 +3,10 @@ const express = require('express');
 const router = express.Router();
 
 const helloWorldController = require('../controllers/hello-world');
+const heroListController = require('../controllers/hero-list');
```

Comment:
I think this is the same problem mentioned by the mentor. List implies the what we will work with at it might change. I would use a different name.

Reply:
thanks, corrected